src/app/shared/components/List/index.tsx

```diff
@@ -204,13 +204,16 @@ ListProps<T, F>): JSX.Element {
   }
 
   return (
-    <div id="generic-list-component" className={`relative flex h-full flex-col overflow-y-hidden ${className}`}>
+    <div
+      id="generic-list-component"
+      className={`relative flex h-full flex-col overflow-x-auto overflow-y-auto ${className}`}
+    >
       {/* HEAD */}
-      <div className="relative flex h-12 flex-shrink-0 flex-row px-5">
+      <div className="sticky top-0 left-0 z-50 flex h-12 w-full flex-shrink-0 flex-row bg-white px-5">
```

Comment:
As "sticky" acts kinda like absolute, it overlaps the breadcrumb options, we'll remove it for now and will fix this issue with the future Component Library

<img width="1204" alt="Screenshot 2023-10-04 at 10 29 36" src="https://github.com/internxt/drive-web/assets/34165099/e8dea76e-0d24-4988-adbe-bb5bae753c38">